core/cache/lxdprofilewatcher.go

```diff
@@ -356,3 +356,8 @@ func (w *MachineLXDProfileWatcher) logError(msg string) {
 	logger.Errorf(msg)
 	w.metrics.LXDProfileChangeError.Inc()
 }
+
+func (w *MachineLXDProfileWatcher) logDebug(msg string) {
```

Comment:
I think we should just call these two lines on site with a comment, and drop the function.
It doesn't feel right to me to be incrementing the error count inside a `logDebug` method.

Reply:
Agreed, I've updated.